sample.py

```diff
@@ -18,9 +18,9 @@
 # message_number_graph_day_list = network.message_number_graph(log_data, nicks, nick_same_list, DAY_BY_DAY_ANALYSIS = True)
 #degree_anal_message_numder = network.degree_analysis_on_graph(message_number_graph)
 # message_time_graph_list = network.create_message_time_graph(log_data, nicks, nick_same_list)
-out_degree_node_number, in_degree_node_number, total_degree_node_number = network.degreeNodeNumberCSV(log_data, nicks, nick_same_list)
```

Comment:
i forgot this in the last PR, correct the name here to follow _ and not camel case